_dactyl/templates/template-sidebar_nav.html

```diff
@@ -4,11 +4,15 @@
         <li class="nav-label">Quick Start</li>
         <li><a href="/overview.html">Overview</a></li>
         <li><a href="/getting-started.html">Getting Started</a></li>
+            <ul>
```

Comment:
I think you'll also need to make these changes to https://github.com/interledger/rfcs/blob/master/tmpl/rfc.ejs.html at the same time so the RFCs themselves serve the updated sidebar.

Reply:
https://github.com/interledger/rfcs/pull/552